libs/domains/observability/feature/src/lib/service-overview/local-chart/local-chart.tsx

```diff
@@ -149,7 +150,7 @@ export const ChartContent = memo(function ChartContent({
 
   return (
     <div className="relative flex h-full">
-      <Chart.Container className="h-full w-full select-none p-5 py-2 pr-0" isLoading={isLoading} isEmpty={isEmpty}>
+      <Chart.Container className="h-full w-full select-none p-5 py-2 pr-5" isLoading={isLoading} isEmpty={isEmpty}>
```

Comment:
Sorry again, could you add a condition to `pr-0` only if `isLoading` and `isEmpty` are false ? It allow to avoid this problem of alignement in this case for example

<img width="691" height="442" alt="Image" src="https://github.com/user-attachments/assets/0863092b-1667-4547-8e17-744bc5423584" />